client/app/lib/onboarding.coffee

```diff
@@ -0,0 +1,111 @@
+# Local class Step
+class Step
```

Comment:
It could be better to put this 'Step' class in a separate step.coffee file, isn't it?